performance/models.py

```diff
@@ -0,0 +1,746 @@
+    contactNumber = Column(BigInteger, nullable=False, index=True)
+    creationDate = Column(DateTime, nullable=False)
+    creator = Column(String(255, u'utf8_bin'), nullable=False)
+    modificationDate = Column(DateTime, nullable=False)
+    modifiedBy = Column(String(255, u'utf8_bin'), nullable=False)
+    owner = Column(String(255, u'utf8_bin'))
+    packType = Column(String(255, u'utf8_bin'))
+    rejectionMessage = Column(String(255, u'utf8_bin'))
+    rejectionReason = Column(String(255, u'utf8_bin'))
+
+
+class NmsSubscriptionPackMessage(Base):
+    __tablename__ = 'nms_subscription_pack_messages'
+
+    id = Column(BigInteger, primary_key=True, index=True)
+    creationDate = Column(DateTime, nullable=False)
+    creator = Column(String(255, u'utf8_bin'), nullable=False)
+    duration = Column(Integer, nullable=False)
+    messageFileName = Column(String(255, u'utf8_bin'))
+    modificationDate = Column(DateTime, nullable=False)
+    modifiedBy = Column(String(255, u'utf8_bin'), nullable=False)
+    owner = Column(String(255, u'utf8_bin'))
+    weekId = Column(String(255, u'utf8_bin'))
+    messages_id_OWN = Column(ForeignKey(u'nms_subscription_packs.id'), index=True)
+    messages_INTEGER_IDX = Column(Integer)
+
+    nms_subscription_pack = relationship(u'NmsSubscriptionPack')
+
+
+class NmsSubscriptionPack(Base):
+    __tablename__ = 'nms_subscription_packs'
+
+    id = Column(BigInteger, primary_key=True, index=True)
+    creationDate = Column(DateTime, nullable=False)
+    creator = Column(String(255, u'utf8_bin'), nullable=False)
+    messagesPerWeek = Column(Integer, nullable=False)
+    modificationDate = Column(DateTime, nullable=False)
+    modifiedBy = Column(String(255, u'utf8_bin'), nullable=False)
+    name = Column(String(100, u'utf8_bin'), unique=True)
+    owner = Column(String(255, u'utf8_bin'))
+    type = Column(String(255, u'utf8_bin'), nullable=False, index=True)
+    weeks = Column(Integer, nullable=False)
+
+
+class NmsSubscription(Base):
+    __tablename__ = 'nms_subscriptions'
+    __table_args__ = (
+        Index('status_endDate_composit_idx', 'status', 'endDate'),
+    )
+
+    id = Column(BigInteger, primary_key=True, index=True)
+    activationDate = Column(DateTime)
+    creationDate = Column(DateTime, nullable=False)
+    creator = Column(String(255, u'utf8_bin'), nullable=False)
+    deactivationReason = Column(String(255, u'utf8_bin'))
+    endDate = Column(DateTime)
+    firstMessageDayOfWeek = Column(String(255, u'utf8_bin'))
+    modificationDate = Column(DateTime, nullable=False)
+    modifiedBy = Column(String(255, u'utf8_bin'), nullable=False)
+    needsWelcomeMessageViaObd = Column(BIT(1), nullable=False)
+    origin = Column(String(255, u'utf8_bin'), nullable=False)
+    owner = Column(String(255, u'utf8_bin'))
+    secondMessageDayOfWeek = Column(String(255, u'utf8_bin'))
+    startDate = Column(DateTime, index=True)
+    status = Column(String(255, u'utf8_bin'), nullable=False, index=True)
```

Comment:
are these always 255 by default or is this model different from what gets generated by MDS? This is more in the context of me wondering if the model will affect the perf testing in any way. I trust you though